Update manage-connections-from-windows-operating-system-components-to… (#9561)
MicrosoftDocs/windows-itpro-docs: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md

For a complete list of the Microsoft Edge policies, see [Available policies for Microsoft Edge](/microsoft-edge/deploy/available-policies).

### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise

> For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies). 

Reviewer: ```suggestion
For a complete list of the Microsoft Edge policies, see [Group Policy and Mobile Device Management (MDM) settings for Microsoft Edge](/microsoft-edge/deploy/available-policies). 
```

> > [!Important]

Reviewer: ```suggestion
> [!IMPORTANT]
```

> > - The following settings are applicable to Microsoft Edge version 77 or later. 

Reviewer: ```suggestion
> - The following settings are applicable to Microsoft Edge version 77 or later. 
```
```suggestion
> - The following settings are applicable to Microsoft Edge version 77 or later.
```

> > - For details on supported Operating Systems see Microsoft Edge supported Operating Systems 

Reviewer: ```suggestion
> - For details on supported Operating Systems, see [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
```

> > - These policies require the Microsoft Edge administrative templates to be applied. For more information on administrative templates for Microsoft Edge see Configure Microsoft Edge policy settings on Windows 

Reviewer: ```suggestion
> - These policies require the Microsoft Edge administrative templates to be applied. For more information on administrative templates for Microsoft Edge, see [Configure Microsoft Edge policy settings on Windows](/deployedge/configure-microsoft-edge).
```

> > - Devices must be domain joined for some of the policies to take effect. 

Reviewer: ```suggestion
> - Devices must be domain joined for some of the policies to take effect.
```

To change how frequently **Windows should ask for my feedback**:

> [!NOTE]

> > Feedback frequency only applies to user-generated feedback, not diagnostic and usage data sent from the device. 

Reviewer 2: This extra space is not required and can be safely omitted.
```suggestion
> Feedback frequency only applies to user-generated feedback, not diagnostic and usage data sent from the device.
```
